content/sql/shop-project-part-2/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+content_type: project
+flavours:
+- none
+prerequisites:
+  hard:
+  - sql/shop-project
+  soft: []
+story_points: 3
```

Comment:
Unless there's a guide on assigning story points, I think you should ignore this.